docs/user/SUMMARY.md

```diff
@@ -17,7 +17,7 @@
     - [Validating metrics](user/metrics/validation-checklist.md)
     - [Error reporting](user/metrics/error-reporting.md)
     - [Metrics collected by the Glean SDKs](user/collected-metrics/metrics.md)
-    - [Enable and Disable metrics using Nimbus](user/metrics/metrics-remote-settings.md)
+    - [Data Control Plane (Remote Metric Configuration)](user/metrics/data-control-plane/index.md)
```

Comment:
you will need to link all the subpages as well.

Reply:
On a side-note...  does [ci-skip] not skip building on Taskcluster?

Reply:
only in the PR title :/

Reply:
But...  it is in the PR title?

Reply:
oh, it must be `[ci skip]` (no dash, https://mozilla.github.io/glean/dev/ci.html#skipping-ci-completely)